include/sound/sof/topology.h

```diff
@@ -37,6 +37,7 @@ enum sof_comp_type {
 	SOF_COMP_SELECTOR,		/**< channel selector component */
 	SOF_COMP_DEMUX,
 	SOF_COMP_ASRC,		/**< Asynchronous sample rate converter */
+	SOF_COMP_VIRT_CON,	/**< Virtual connection, sent by the VirtIO guest */
```

Comment:
should we move this to e.g. 0xFFFF to avoid any confusion with firmware components?

This is not known to the firmware so should not get in the way of firmware-managed components